staging/src/k8s.io/client-go/tools/clientcmd/validation_test.go

```diff
@@ -385,79 +385,76 @@ func TestValidateAuthInfoExec(t *testing.T) {
 	test.testConfig(t)
 }
 
```

Comment:
the moving functions around makes it more difficult than needed to review.

Reply:
agree... please put these back in the previous order so the diff reflects what is actually being changed